src/OrchardCore.Modules/OrchardCore.Google/Views/GoogleAnalyticsSettings.Edit.cshtml

```diff
@@ -0,0 +1,27 @@
+@using OrchardCore.Google.Analytics.ViewModels;
+@model GoogleAnalyticsSettingsViewModel;
+
+<p class="alert alert-warning">@T["The current tenant will be reloaded when the settings are saved."]</p>
```

Comment:
The tenant doesn't need to be restarted for sure settings that are not later in Configuration